sample/src/main/java/no/finntech/rxactivityresponse/sample/RxLoginExample.java

```diff
@@ -0,0 +1,182 @@
+import rx.android.schedulers.AndroidSchedulers;
+import rx.functions.Action1;
+import rx.schedulers.Schedulers;
+
+public class RxLoginExample extends Button implements View.OnClickListener {
+    private static final String GOOGLE_PLUS_SCOPES = Scopes.PLUS_LOGIN + " " + "email";
+    private ResponseHandler responseHandler = new ResponseHandler();
+
+    public RxLoginExample(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        setOnClickListener(this);
+    }
+
+    private static class GoogleLoginTransformer implements Observable.Transformer<GoogleApiClient, String> {
+        private static Handler handler = new Handler(Looper.getMainLooper());
+        private final Activity activity;
+        private final ResponseHandler responseHandler;
+
+        public GoogleLoginTransformer(Activity activity, ResponseHandler responseHandler) {
+            this.activity = activity;
+            this.responseHandler = responseHandler;
+        }
+
+        @Override
+        public Observable<String> call(Observable<GoogleApiClient> o) {
+            return o.observeOn(Schedulers.io())
+                    .lift(new Observable.Operator<String, GoogleApiClient>() {
+                              @Override
+                              public Subscriber<? super GoogleApiClient> call(final Subscriber<? super String> subscriber) {
+                                  final Subscriber<GoogleApiClient> s = new Subscriber<GoogleApiClient>() {
+                                      @Override
+                                      public void onCompleted() {
+                                          subscriber.onCompleted();
+                                      }
+
+                                      @Override
+                                      public void onError(Throwable throwable) {
+                                          subscriber.onError(throwable);
+                                      }
+
+                                      @Override
+                                      public void onNext(GoogleApiClient client) {
+                                          String accountName = Plus.AccountApi.getAccountName(client);
+                                          Account account = new Account(accountName, GoogleAuthUtil.GOOGLE_ACCOUNT_TYPE);
+                                          String scopes = "oauth2:" + GOOGLE_PLUS_SCOPES;
+                                          try {
+                                              subscriber.onNext(GoogleAuthUtil.getToken(activity.getApplicationContext(), account, scopes));
+                                              subscriber.onCompleted();
+                                          } catch (IOException e) {
+                                              subscriber.onError(e);
+                                          } catch (final GoogleAuthException e) {
+                                              if (e instanceof UserRecoverableAuthException) {
```

Comment:
I think it would be nicer to catch these in increasingly generic order instead of the instanceof check, but that's probably just a matter of taste.